.github/workflows/fe-code-quality.yml

```diff
@@ -1,14 +1,6 @@
 name: Frontend Code Quality Workflow
 
-on:
-  workflow_call:
-    inputs:
-      GHA_REF:
-        type: string
-        required: true
-      GHA_BASE:
-        type: string
-        required: true
+on: workflow_call
```

Comment:
why do we need this?

Reply:
That's a revert back to the original state before yours.